docs/revsets.md

```diff
@@ -34,9 +34,9 @@ typically only one visible commit with a given change ID). A unique prefix of
 the full change ID can also be used. It is an error to use a non-unique prefix.
 
 Use double quotes to prevent a symbol from being interpreted as an expression.
-For example, `"x-1"` is the symbol `x-1`, not the parents of symbol `x`.
+For example, `"x-"` is the symbol `x-`, not the parents of symbol `x`.
 Taking shell quoting into account, you may need to use something like
-`jj log -r '"x-1"'`.
+`jj log -r '"x-"'`.
```

Comment:
Not sure this section makes sense now, "x-" isn't actually a valid expression, is it?

Reply:
It is valid - it means "parents of x" (see https://github.com/martinvonz/jj/blob/main/docs/revsets.md#operators)